Tests/LibWeb/Ref/do-not-scroll-overflow.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
```

Comment:
`do-not-scroll-overflow.html` test name sounds a bit confusing. could it be more descriptive, like `box-without-scrollable-overflow-should-not-consume-events.html`?